lottie/lib/lottiestatus.h

```diff
@@ -49,12 +49,11 @@ class LottieStatus final : public QObject {
     m_totalTime = 0;
     m_error = true;
     m_errorString = errorString;
-    emit changed(m_playing, m_currentTime, m_totalTime, m_error, m_errorString);
+    emit changed();
   }
 
  signals:
-  void changed(bool playing, double currentTime, int totalTime, bool error,
-               const QString& errorString);
+  void changed();
```

Comment:
So while at it i'm building against qt6.9-beta2 and moc throws an error here, as we never read this.